src/handlers/hello-world.ts

```diff
@@ -54,6 +54,31 @@ export async function helloWorld(context: Context) {
     }
   }
 
+  if (customStringsUrl) {
+    try {
+      const response = await fetch(customStringsUrl).then((value) => value.json());
+      await octokit.issues.createComment({
+        owner: payload.repository.owner.login,
+        repo: payload.repository.name,
+        issue_number: payload.issue.number,
+        body: response.string,
+      });
+    } catch (error) {
+      /**
+       * logger.fatal should not be used in 9/10 cases. Use logger.error instead.
+       *
+       * Below are examples of passing error objects to the logger, only one is needed.
+       */
+      if (error instanceof Error) {
+        logger.error(`Error creating comment:`, { error: error, stack: error.stack });
+        throw error;
+      } else {
+        logger.error(`Error creating comment:`, { err: error, error: new Error() });
```

Comment:
Is there a way to use a single/unified instance of `logger.error` instead of multiple calls:
```
logger.error(`Error creating comment:`, { error: error, stack: error.stack });
logger.error(`Error creating comment:`, { err: error, error: new Error() });
```

?